platform/android/java/build.gradle

```diff
@@ -141,12 +163,7 @@ task generateGodotTemplates(type: GradleBuild) {
             && targetLibs.listFiles() != null
             && targetLibs.listFiles().length > 0) {
             String capitalizedTarget = target.capitalize()
-            // Copy the generated aar library files to the custom build directory.
-            tasks += "copy" + capitalizedTarget + "AARToAppModule"
-            // Copy the generated aar library files to the bin directory.
-            tasks += "copy" + capitalizedTarget + "AARToBin"
-            // Copy the prebuilt binary templates to the bin directory.
-            tasks += "copy" + capitalizedTarget + "BinaryToBin"
+            tasks += "generate" + capitalizedTarget + "TemplateRun"
```

Comment:
@thebestnom Looks like the logic update removes the `release_debug` generated templates, which in this context would be the same as the `debug` target but with `doNotStrip` set to `false`.

@akien-mga I'd imagine we still want the `release_debug` templates (with stripped symbols) for stable releases, so we may need to differentiate between generated `debug` templates with symbols (i.e: scons `debug` target) and generated `release_debug` templates without symbols (e.g: scons `release_debug` target).

Reply:
Yes, we still need a `scons target=release_debug` template with symbols stripped which is used as the official "debug" template that users use (to debug their project, not the engine itself).

That's why I said that it might be better to split this in three explicit targets so that engine developers can use e.g. `./gradlew generateDevTemplate` to get `target=debug` template which is not stripped.

Reply:
We can roll most of this back and just use -PdoNotStrip instead

Reply:
@thebestnom I think you can revert `generateGodotTemplates` so it behaves as it does currently, and rename `generateDebugTemplate`  to `generateDevTemplate` as @akien-mga suggested.
Then we'll just update the instructions to indicate that generating a debug build with symbols can be done with the `./gradlew generateDevTemplate` command.